journal/updated/09-chapter-8.adoc

```diff
@@ -195,50 +199,50 @@ and by me rather desirable that it may soon be experienced.
 
 On the 17th of first month, 1785, we left Bristol,
 having the blessed reward of pure peace for our labours.
-Lydia Hawksworth accompanied us to Frenchay, where we had a meeting;
-several Friends from the city were there, and we were owned by our great Lord,
-his name was magnified, and his own works did praise him.
+Lydia Hawksworth accompanied us to Frenchay, where we had a meeting.
+Several Friends from the city were there, and we were acknowledged by our great Lord;
+His name was magnified, and His own works did praise Him.
 
-Early on the following morning, accompanied by Thomas Scantlebury, we went to Painswick,
-and were at their meeting on the 19th; and at Tewkesbury on the 21st.
+Early the following morning, accompanied by Thomas Scantlebury, we went to Painswick,
+and were at their meeting on the 19th, and at Tewkesbury on the 21st.
 
-On seventh-day we reached Worcester, and lodged at our kind friend Timothy Bevington`'s:
-were at their meetings on first-day, where were also Patience Brayton and her companion.
+On seventh-day we reached Worcester, and lodged at our kind friend Timothy Bevington`'s.
+Were at their meetings on first-day, where were Patience Brayton and her companion also.
 On second-day, the 24th, my dear companion left me, she being very desirous to get home;
-but I was not easy to leave Worcester so soon, and stayed more than a week longer,
+but I was not clear to leave Worcester so soon, and stayed more than a week longer,
 in which time I visited several of my acquaintance, had several satisfactory seasons,
 and was much favoured at some of their public meetings,
 so that I came away with sweet peace.
 
 On fourth-day morning I took an affectionate farewell of my dear friends,
 and was accompanied by George Beckett and Mary Bewley to Alcester, where they left me,
 and I lodged at the widow Stanley`'s,
-from which I was favoured with the company of an agreeable young woman, Mary Ford,
+from which place I was favoured with the company of an agreeable young woman, Mary Ford,
 to Jeffery Bevington`'s, at Eddington.
-I was at their meeting on fifth-day; and on sixth, at the meeting at Ormscott;
+I was at their meeting on fifth-day, and at the meeting at Ormscott on sixth,
 but the weather being severe, there were few Friends present;
 yet it was a time of deep travail and hard labour for the honest-hearted.
 On fourth-day I was at the meeting at Warwick, and on the following day reached Coventry,
 where I was joined by my husband.
-Attended their meeting on first-day, Timothy Bevington of Worcester, being there also,
-and we were owned by our Great Master, and employed in his service.
-On the 15th of second month we got safe home, and found our dear children well.
+Attended their meeting on first-day, Timothy Bevington of Worcester being there also,
+and we were acknowledged by our Great Master, and employed in His service.
```

Comment:
I think that if we're going to switch owned to acknowledged, it is often best to say "acknowledged by the presence of..."